admin/app/controllers/solidus_admin/roles_controller.rb

```diff
@@ -63,6 +98,10 @@ def destroy
 
     private
 
+    def find_role
```

Comment:
nit: thoughts on sticking to the convention used in rails scaffold code and use `set_role` here?

_from `rails g scaffold`_

```rb

  private
    # Use callbacks to share common setup or constraints between actions.
    def set_post
      @post = Post.find(params[:id])
    end

    # Only allow a list of trusted parameters through.
    def post_params
      params.require(:post).permit(:title, :body)
    end
```

Reply:
Great call, yes, let's do it in one pass for everything 👍

Reply:
Sounds good, I can put that bulk rename change up today.